addons/mail/static/tests/qunit_suite_tests/components/link_preview_tests.js

```diff
@@ -0,0 +1,306 @@
+        og_title: 'video title',
+        og_type: 'video.other',
+        source_url: 'https://www.odoo.com',
+    });
+    const mailChannelId = pyEnv['mail.channel'].create({});
+    pyEnv['mail.message'].create({
+        body: 'not empty',
+        link_preview_ids: [linkPreviewId],
+        message_type: 'comment',
+        model: "mail.channel",
+        res_id: mailChannelId,
+    });
+    const { openDiscuss } = await start({
+        discuss: {
+            params: {
+                default_active_id: `mail.channel_${mailChannelId}`,
+            },
+        },
+    });
+    await openDiscuss();
+
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewVideo',
+        "should have link preview video in DOM"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewVideo_title',
+        "Should display the link preview title"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewVideo_description',
+        "Link preview should show the link description"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_linkPreviewVideo_overlay',
+        "Should display overlay inside the link preview video image"
+    );
+
+    await afterNextRender(() => {
+        document
+            .querySelector('.o_LinkPreviewVideo')
+            .dispatchEvent(new window.MouseEvent('mouseenter'));
+    });
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewAside',
+        'Should contain a link preview aside'
+    );
+});
+
+QUnit.test('Link preview image layout', async function (assert) {
+    assert.expect(1);
+
+    const pyEnv = await startServer();
+    const linkPreviewId = pyEnv['mail.link.preview'].create({
+        og_image: 'https://upload.wikimedia.org/wikipedia/commons/thumb/4/41/Siberischer_tiger_de_edit02.jpg/290px-Siberischer_tiger_de_edit02.jpg',
+        og_title: 'https://upload.wikimedia.org/wikipedia/commons/thumb/4/41/Siberischer_tiger_de_edit02.jpg/290px-Siberischer_tiger_de_edit02.jpg',
+        og_type: 'image',
+        source_url: 'https://upload.wikimedia.org/wikipedia/commons/thumb/4/41/Siberischer_tiger_de_edit02.jpg/290px-Siberischer_tiger_de_edit02.jpg',
+    });
+    const mailChannelId = pyEnv['mail.channel'].create({});
+    pyEnv['mail.message'].create({
+        body: 'not empty',
+        link_preview_ids: [linkPreviewId],
+        message_type: 'comment',
+        model: "mail.channel",
+        res_id: mailChannelId,
+    });
+    const { openDiscuss } = await start({
+        discuss: {
+            params: {
+                default_active_id: `mail.channel_${mailChannelId}`,
+            },
+        },
+    });
+    await openDiscuss();
+
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewImage',
+        "should have link preview image"
+    );
+});
+
+QUnit.test('Remove link preview', async function (assert) {
```

Comment:
Prefer naming test like assertion comment, so that it's very clear what is expected for this test.
For example, "link preview card should be deletable"